.github/workflows/run-docs-generation.yml

```diff
@@ -93,6 +93,12 @@ jobs:
           # If generating docs for more providers here, be sure to update
           # the description of the draft PR that is opened in the next step.
           pushd docs
+
+          pushd tools/resourcedocsgen
+          go mod edit -replace github.com/pulumi/pulumi/pkg/v3=../../../pulumi/pkg
```

Comment:
I did this because I plan to pin the versions used by the `resourcedocsgen` tool in `docs`. For the sake of this workflow, it does need the local override so that the templates used by it are what's in the PR current branch always.